### PR TITLE
MM-T1437 Try to create a new team using restricted words

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary

- Added end to end test to verify reserved words cannot be used to create teams

#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T1437

#### Related Pull Requests

None

#### Screenshots

N/A

### CODE REVIEW

File: e2e/cypress/integration/team_settings/create_a_team_spec.js [hunk 1]
```diff
@@ -52,4 +45,55 @@ describe('Teams Suite', () => {
...
     });
+
+    it('MM-T1437 Try to create a new team using restricted words', () => {
+        openTeamCreationModal()
+
+        // # Input team name as Team Test
+        cy.get('#teamNameInput').should('be.visible').type('plugins');
+
+        // # Click Next button
+        cy.findByText('Next').should('be.visible').click();
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/create_a_team_spec.js:56]: Could you please move these and the last step to close the modal to `tryReservedTeamURLAndVerifyError`?

File: e2e/cypress/integration/team_settings/create_a_team_spec.js [hunk 2]
```diff
@@ -52,4 +46,56 @@ describe('Teams Suite', () => {
...
+        // # Click "Create a Team"
+        cy.findByText('Create a Team').should('be.visible').click();
+    });
+}
+
+function tryReservedTeamURLAndVerifyError(teamURL) {
+    // # Open the create team fullscreen modal from hamburger menu
+    openTeamCreationModalFromHamburgerMenu();
+
+    // # Input a random team name
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/create_a_team_spec.js:79]: "random" does seemed accurate.

File: e2e/cypress/integration/team_settings/create_a_team_spec.js [hunk 3]
```diff
@@ -52,4 +46,56 @@ describe('Teams Suite', () => {
...
+    cy.get('#teamNameInput').should('be.visible').type('team-name-123');
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/create_a_team_spec.js:80]: I'm good as is but passing `teamURL` or team name will work too.
```suggestion
    cy.get('#teamNameInput').should('be.visible').type(teamURL);
```

File: package-lock.json [hunk 4]
```diff
@@ -14313,6 +14316,12 @@
         "postcss": "^7.0.14"
       }
     },
+    "icu4c-data": {
+      "version": "0.67.2",
+      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.67.2.tgz",
+      "integrity": "sha512-OIRiop+k1IVf4TBLEOj910duoO9NKwtJLwp++qWT6KT5gRziHNt+5gwhcGuTqRy++RTK2gLoAIbk8KYCNxW++g==",
+      "dev": true
+    },
```
Comment (saturninoabril) [inline package-lock.json:14324]: Please revert changes here.

File: e2e/cypress/integration/team_settings/create_a_team_spec.js [hunk 5]
```diff
@@ -52,4 +46,56 @@ describe('Teams Suite', () => {
         // * Team name should displays correctly at top of LHS
         cy.get('#headerTeamName').should('contain', 'Team Test');
     });
+
+    it.only('MM-T1437 Try to create a new team using restricted words', () => {
```
Comment (srkgupta) [inline e2e/cypress/integration/team_settings/create_a_team_spec.js:50]: Can you revert the `only` block from this test.

**Pull request merged**